src/Persistence/ManagerPersistenceService.py

```diff
@@ -1,111 +1,203 @@
 
         try:
-            cur.execute("SELECT fname, lname, projectName FROM project, team, employee WHERE employee.employeeID = teamEmployeeID AND ")
-            # FIXME store the query results into data objects
+            # TODO wait for testing the sql query
+            cur.execute("SELECT project.projectID, project.projectName, project.currentTeamID, project.projectStatus FROM project, team, employee WHERE team.teamID = project.currentTeamID AND team.teamID = employee.teamID AND employee.employeeID = '%(1)s' AND employee.teamID = '%(2)s';" % {"1": teamEmployeeID, "2": teamID})
+
+            records = cur.fetchall()
+            projectList = []
+            for row in records:
+                projectInstance = Project(row[0], row[1], row[2], row[3])
+
+                projectList.append(projectInstance)
 
         except mysql.connector.Error as error:
             print(error)
             sys.exit(1)
         dbConnection.connection.close()
 
+        return projectList
+
     """
     Assign Employee to team
     """
     @staticmethod
-    def assignEmployeeToTeam(teamEmployeeId, managerId):
+    def assignEmployeeToTeam(teamEmployeeId, managerId, teamID):
         cur = dbConnection.connection.cursor()
 
         try:
-            # FIXME wrong query so far, will fix it later
-            cur.execute("INSERT INTO team () '%(1)s';" % {"1": managerId})
-            # FIXME store the query results into data objects
+            # remove the employee from current team
+            cur.execute("DELETE FROM teamMembers WHERE employeeID = '%(1)s';" % {"1": teamEmployeeId})
+            dbConnection.connection.commit()
+
+            # update the team information to the employee table
+            cur.execute("UPDATE employee SET teamID = '%(1)s' WHERE employeeID = '%(2)s';" % {"1": teamID, "2": teamEmployeeId})
+            dbConnection.connection.commit()
 
+            # insert the employee information to the teamMembers table
+            cur.execute("INSERT INTO teamMembers VALUES ('%(1)s', '%(2)s');" % {"1": teamID, "2": teamEmployeeId})
+            dbConnection.connection.commit()
         except mysql.connector.Error as error:
             print(error)
             sys.exit(1)
         dbConnection.connection.close()
 
+        print("The employee (" + teamEmployeeId + ") has been added to the team (" + teamID + "), which managed by the Manager (" + managerId + ").")
+
     """
     Remove Employee from team
     """
     @staticmethod
-    def removeEmployeeFromTeam(teamEmployeeId):
-        pass
+    def removeEmployeeFromTeam(teamID, teamEmployeeId):
+        cur = dbConnection.connection.cursor()
+
+        try:
+            cur.execute("DELETE FROM teamMembers WHERE teamID = '%(1)s' AND employeeID = '%(2)s';" % {"1": teamID, "2": teamEmployeeId})
+            dbConnection.connection.commit()
+            cur.execute("UPDATE employee SET teamID = 'NULL' WHERE employeeID = '%(1)s';" % {"1": teamEmployeeId})
```

Comment:
I think NULL should be unquoted https://stackoverflow.com/questions/9314353/set-value-to-null-in-mysql